### PR TITLE
made website responsive

### LINKED ISSUE #29: Make the website responsive
Go to the mobile view of the website and you can see how the search and add title buttons appear dis-oriented. Fix that. Note that the desktop view should not be affected.

### PR DESCRIPTION
**Pull request by blaze-assault**
__29__ 

Fixes #29

#### Short description of what this resolves:
made website responsive.

#### Changes proposed in this pull request and/or Screenshots of changes:

-
-
-

### CODE REVIEW

**Comment by abhishekmsingh** [comment]
Give me a live link @blaze-assault

**Comment by abhishekmsingh** [comment]
For generating live link,
go to your repository settings, scroll down to github pages section and select master branch in the sources. The url you get just give me that.

**Comment by blaze-assault** [comment]
Ok sir .i will do ot .give me some time.

**Comment by blaze-assault** [comment]
> Give me a live link @blaze-assault

https://blaze-assault.github.io/pages/index.html

@abhiandthetruth

**Comment by abhishekmsingh** [comment]
@blaze-assault The search text-box appears to be odd. You changed the class from form-control to search-small, which lead to removal of bootstrap styling. Try changing it to 'search-small form-control'.

**Comment by blaze-assault** [comment]
at first without correction the search button came below search box and
they were not aligned .i made them in line in mobile view and it looked
fine.please clarify what is looking odd and i will do it by tommorrow.

File: pages/index.html [hunk 1]
```diff
@@ -28,11 +28,16 @@
       <div class="navbar-header">
         <a href="index.html" class="navbar-brand"><h1>Libook</h1></a>
       </div>
+      <div class="row">
       <form action="#" class="form-inline">
-        <input id="search-text" name= "search-text" type="text" class="form-control" placeholder="Search">
-        <button id="search-btn" class="btn btn-outline-success ml-2" type="submit">Search</button>
+     
+        <div class=" col-xs-3 " > <input id="search-text" class="search-small" name= "search-text" type="text" class="form-control" placeholder="Search"></div>
```
Comment (abhishekmsingh) [inline pages/index.html:34]: here the class has been changed from form-control to search-small. Try using 'search-small form-control'
image

**Comment by blaze-assault** [comment]
I got it what you are saying sir.i will fix it by tommorrow .

**Pull request closed without merging**